### PR TITLE
G1ul10 flying

### PR DESCRIPTION
**Pull request by g1ul10**
Add flying attribute to Forest Imp, Spitting Drake, Sun Demon and Wind Demon and in general use the icon to represent this attribute.

### CODE REVIEW

**Comment by GinoGalotti** [comment]
Looks cool to me, thanks for the edit!

Just one question... any way to double check that Forest Imp and Spitting Drake should have flying? It makes sense, but we based this on top of https://docs.google.com/spreadsheets/d/1Z7JLWHzaXJsuaA-Up-XpmmZzwz-VkJOHY4gmkmrBSs4/edit#gid=633341686 ; and it doesn't say that they fly. I don't have the game myself... so if you can double check, I'll trust you! (And I'll merge!)

**Comment by g1ul10** [comment]
Yes they do. I own the game. If you are not scared by some spoiler, you can check the monster descriptions here http://www.cephalofair.com/gloomhaven-monster-spoilers. That source is not completely accurate, though. By the way, I forgot to add the flying attribute to "Living Spirits". They are flying too. I'll prepare another pull request after this one or you can fix it yourself, as you prefer.

**Comment by GinoGalotti** [comment]
Nah, feel free to just commit that to the branch, push it, and I'll merge.
Usually, for this kind of changes we rebase and squash everything in 1 commit. Feel free to do it! (I think if I click "Squash and Merge" it'll lose ownership and lose your face on the commit.

In case you're not familiar with git, just running `git rebase -i upstream/master' should be enough. You'll have the list of all your commits, and you can squash them in one.

File: monster_stats.js [hunk 1]
```diff
@@ -3493,7 +3493,7 @@ MONSTER_STATS = {
             "attack": 2,
             "range": 2,
             "attributes": [
-              "%shield% 1"
+              "%flying%","%shield% 1"
```
Comment (g1ul10) [inline monster_stats.js]: No particular reason I can remember. I probably did not put enough attention to implicit syntax convention. Apologies.

**Pull request merged**